### PR TITLE
Add data-bound setters for View

### PR DESCRIPTION
**Pull request by Vlatombe**
This allows programmatic edit of View.

While editing, I noticed that the `help` attribute were misplaced, causing them not to be displayed, so I fixed that as well.

See [JENKINS-XXXXX](https://issues.jenkins.io/browse/JENKINS-XXXXX).

### Proposed changelog entries

- Developer: Added setters for `View#filterExecutor` and `View#filterQueue`
- Fix missing help sections for view filter executor and queue fields.

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [X] The Jira issue, if it exists, is well-described.
- [X] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [X] There is automated testing or an explanation as to why this change has no tests.
- [X] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [X] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [X] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/8511"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/model/View/configure.jelly
Comment (jglick) [inline core/src/main/resources/hudson/model/View/configure.jelly:46]: Why would you need to set `name` when the entry already has `field`?

**Review by jglick — COMMENTED** [review]
Check whether https://github.com/jenkinsci/jenkins/blob/13032e1e05b116fe9d6d063ff7cd8a92b8a03c41/core/src/main/java/hudson/model/View.java#L1033-L1034 can now be removed.

File: core/src/main/resources/hudson/model/View/configure.jelly
Comment (Vlatombe) [reply core/src/main/resources/hudson/model/View/configure.jelly:46]: was already there, and I think the parameter name is `_.filterQueue` when using `field` only.

**Review by jglick — APPROVED** [review]
Closer to standard databinding idioms.

**Comment by jglick** [comment]
```
org.htmlunit.ElementNotFoundException: elementName=[textarea] attributeName=[name] attributeValue=[description]
	at org.htmlunit.html.HtmlForm.getTextAreaByName(HtmlForm.java:738)
	at hudson.model.ViewDescriptorTest.invisiblePropertiesOnViewShoudBePersisted(ViewDescriptorTest.java:123)
```

https://github.com/jenkinsci/jenkins/blob/2fc718da2a377d9dc411ebae95badf2f42212bef/test/src/test/java/hudson/model/ViewDescriptorTest.java#L123 presumably needs to be `_.description` now?

**Comment by timja** [comment]
> ```
> org.htmlunit.ElementNotFoundException: elementName=[textarea] attributeName=[name] attributeValue=[description]
> 	at org.htmlunit.html.HtmlForm.getTextAreaByName(HtmlForm.java:738)
> 	at hudson.model.ViewDescriptorTest.invisiblePropertiesOnViewShoudBePersisted(ViewDescriptorTest.java:123)
> ```
> 
> https://github.com/jenkinsci/jenkins/blob/2fc718da2a377d9dc411ebae95badf2f42212bef/test/src/test/java/hudson/model/ViewDescriptorTest.java#L123
> 
> presumably needs to be `_.description` now?

can you run through ATH to be safe please?

**Comment by Vlatombe** [comment]
Filed https://github.com/jenkinsci/acceptance-test-harness/pull/1372 (the method wasn't used already so added a test)

**Review by timja — APPROVED** [review]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

File: core/src/main/resources/hudson/model/View/configure.jelly
Comment (jglick) [reply core/src/main/resources/hudson/model/View/configure.jelly:42]: Yes, see 33209a798a568a9dfd5a1ddc507574c024b650cb for example.

**Pull request merged**
